document-store/src/test/java/org/hypertrace/core/documentstore/postgres/PostgresCollectionTest.java

```diff
@@ -106,20 +108,22 @@ void testUpdateAtomicWithFilter() throws IOException, SQLException {
     final PreparedStatement update2PrepStatement = mock(PreparedStatement.class);
     when(mockConnection.prepareStatement(
             String.format(
-                "UPDATE %s SET document=jsonb_set(document, ?::text[], to_jsonb(?)) WHERE id=?",
+                "UPDATE %s SET document=jsonb_set(COALESCE(document, '{}'), ?::text[], to_jsonb(?)) WHERE id=?",
```

Comment:
Did we find the case where `document` is null?

Reply:
Nope. In fact, it is a non-null column. But, it is just there to be consistent with other (nested) fields.